src/firejail/main.c

```diff
@@ -1110,6 +1110,13 @@ int main(int argc, char **argv, char **envp) {
 	if (check_arg(argc, argv, "--quiet", 1) || (env_quiet && strcmp(env_quiet, "yes") == 0))
 		arg_quiet = 1;
 
+	// process --debug
+	const char *env_debug = env_get("FIREJAIL_DEBUG");
```

Comment:
```suggestion
```

It looks like the env var would end up overriding a CLI argument (`--quiet`),
so remove this for now.

Also, I'm not sure why, but this variable is currently only set and never read.
It was added on commit ccd01529a ("fix #2820 - adjustable file copy limit;
export FIREJAIL_DEBUG into sbox", 2019-07-04).

Anyway, I have an idea of how to refactor this later to include the env var and
improve the handling of both quiet/debug (it would involve more code, while the
fix in this PR is rather straightforward).

Reply:
> It looks like the env var would end up overriding a CLI argument (--quiet),
> so remove this for now.

How does it override it in this case?

Also if we remove `env_debug` we also need to remove the `||` statement on the next like that uses `env_debug`.

Reply:
> > It looks like the env var would end up overriding a CLI argument (--quiet),
> > so remove this for now.
> 
> Ah you mean because it would set `arg_quiet` to 0

Yes.

> but if somebody wants to enable debugging, isn't that expected?

The usual convention on Linux etc is that CLI options always override their
corresponding environment variables, probably because the options are usually
easier to type and may have shell completion.  Also, the variables may come
from ~/.bashrc or somewhere not immediately obvious during usage and are
probably less likely to be well-documented compared to CLI options.

Personally I'd find it surprising if the override did not work this way in a
given tool.

> Also if we remove `env_debug` we also need to remove the right side of the
> `||` statement on the next line that uses `env_debug`.

Yes, good catch.

Reply:
> The usual convention on Linux etc is that CLI options always override their
corresponding environment variables, probably because the options are usually
easier to type and may have shell completion. Also, the variables may come
from ~/.bashrc or somewhere not immediately obvious during usage and are
probably less likely to be well-documented compared to CLI options.

Ah I see, makes sense.

> Yes, good catch.

I'll force push with the new changes and resolve the conversation.